ui/src/router/components/form/validation/schema.js

```diff
@@ -103,9 +105,22 @@ const routeSchema = yup.object().shape({
   type: yup.string().oneOf(["PROXY"], "Route Type is required"),
   endpoint: yup
     .string()
-    .required("Valid url is required")
-    .url("Valid url is required"),
+    .when('$protocol', {
+      is: "UPI_V1",
+      then: (schema) => schema
+      .required("Valid grpc endpoint is required eg. {url}:{port}")
```

Comment:
It should be `{host}:{port}`, `url` implies `scheme` should be part of the input.